Src/AmrCore/AMReX_AmrMesh.cpp

```diff
@@ -437,6 +448,41 @@ AmrMesh::LevelDefined (int lev) noexcept
     return lev <= max_level && !grids[lev].empty() && !dmap[lev].empty();
 }
 
+DistributionMapping
+AmrMesh::MakeDistributionMap (int lev, BoxArray const& ba)
+{
+
+    BL_PROFILE("AmrMesh::MakeDistributionMap()");
+
+    // useful print statement to notify that a new distribution map is being created at lev
+    amrex::Print() << "creating new distribution map on level: " << lev + 1 << "\n";
```

Comment:
```suggestion
    if (verbose) {
        amrex::Print() << "Creating new distribution map on level: " << lev << "\n";
    }
```
Don't want to print this if verbose is 0. Also let's use the C++ convention that level numbering starts with 0.